src/OpenCL.cpp

```diff
@@ -127,14 +127,14 @@ __kernel void merge(
                         __global const net_t * restrict in,
                         __global net_t * restrict out,
                         __private const int channels) {
-        // cl::NDRange global(outputs, 19*19);
+        // cl::NDRange global(outputs, BOARD_SIZE*BOARD_SIZE);
```

Comment:
BOARD_SQUARES

Reply:
In OpenCL, I haven't defined BOARD_SQUARES : it seems not that useful to me given the only place it can be used is in this one commented line (Only place in OpenCL where there's a BOARD_SIZE*BOARD_SIZE). 

Do you really want me to replace this by BOARD_SQUARE?